applications/frontier/Makefile

```diff
@@ -0,0 +1,57 @@
+PROJECT = frontier
+ROOT = ../..
+REBAR = $(ROOT)/utils/rebar/rebar
+DIALYZER = dialyzer
+
+EBINS = $(shell find $(ROOT)/core -maxdepth 2 -name ebin -print) $(shell find $(ROOT)/deps -maxdepth 2 -name ebin -print)
```

Comment:
See app Makefiles like Crossbar; they've been re-worked to be a bit more efficient in including ebin/ paths that are actually necessary for compilation (rather than everything like this does).